org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Scannable.java

```diff
@@ -46,7 +46,7 @@
 	private final static Logger logger = LoggerFactory.getLogger(_Scannable.class);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)